unittests/AST/ASTImporterTest.cpp

```diff
@@ -4593,6 +4593,25 @@ TEST_P(ASTImporterOptionSpecificTestBase,
   EXPECT_EQ(0u, ToTU->getASTContext().getDiagnostics().getNumWarnings());
 }
 
+TEST_P(ASTImporterOptionSpecificTestBase, EnumsInAnonNamespace) {
```

Comment:
Is this test related to the brace error?

Reply:
Brace range is not imported correctly, the new test reveals this problem. (Only import of EnumDecl is changed so the changes are related from this point of view. The BraceRange import is tested only with the new test.)